packages/react-component-library/src/components/NumberInput/useInputText.ts

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useRef, useState } from 'react'
+import { isFinite } from 'lodash'
+
+import { UnitPosition } from './NumberInput'
+import { UNIT_POSITION } from './constants'
+
+const EXTRA_SPACING = 3
+
+function getTextWidth({
```

Comment:
Clever technique! I assume it plays nice cross browser?

Reply:
This is IE 11..

![number-input-ie11](https://user-images.githubusercontent.com/56078793/95450956-037d0480-095f-11eb-969e-a13a9af23841.gif)